Add Handbook Process folder and add first process (#41)
giantswarm/handbook: content/docs/support-and-ops/processes/customer-check-upgrade-changes.md

---
title: "How to check for changes in k8s upgrades"
owner:
- https://github.com/orgs/giantswarm/teams/team-teddyfriends
confidentiality: public
---

### Purpose

> When uprading Giant Swarm releases you will notice that the kubernetes version is also being upgraded. As part of the upgrade process it should be a habit to check for the newest changes upstream. This ensures that the diffs, deprecations, and new features from an upstream kubernetes perspective are known. 

Reviewer: ```suggestion
When upgrading Giant Swarm releases you will notice that the kubernetes version is also being upgraded. As part of the upgrade process it should be a habit to check for the newest changes upstream. This ensures that the diffs, deprecations, and new features from an upstream kubernetes perspective are known. 
```

> When upgrading Giant Swarm releases you will notice that the kubernetes version is also being upgraded. As part of the upgrade process it should be a habit to check for the newest changes upstream. This ensures that the diffs, deprecations, and new features from an upstream kubernetes perspective are known. 

Reviewer 2: ```suggestion
When upgrading Giant Swarm releases you will notice that the Kubernetes version is also being upgraded. As part of the upgrade process, it should be a habit to check for the newest changes upstream. This ensures that the diffs, deprecations, and new features from an upstream Kubernetes perspective are known to the customer. 
```

> There is not a right or wrong way to go about this process because there is not one tool that simply gives all of the answers. However, the following tools combine to give a nice overview of the important changes. Let us know if you have thoughts as well!  

Reviewer 2: ```suggestion
There is not a right or wrong way to go about this process because there is no single tool that simply gives all of the answers. However, the following tools combine to give a nice overview of the important changes. Let us know if you have thoughts or recommended tools as well!  
```
